boards/components/src/console.rs

```diff
@@ -113,20 +113,31 @@ macro_rules! console_component_static {
         let console = static_buf!(Console<'static>);
         (write_buf, read_buf, console_uart, console)
     }};
```

Comment:
Can you make this branch an invocation of the other? That way we don't have to duplicate as much business logic in our macros.

Reply:
good idea, fixed!